ikarus/python/test/linearelastictest.py

```diff
@@ -125,14 +125,10 @@ def fixLeftHandEdge(vec, localIndex, localView, intersection):
     indexSet = grid.indexSet
 
     stressFuncScalar = grid.function(
-        lambda e, x: fes[indexSet.index(e)].calculateAt(
-            req, x, iks.ResultType.linearStress
-        )[0]
+        lambda e, x: fes[indexSet.index(e)].calculateAt(req, x, "linearStress")[0]
```

Comment:
This has to be also a MultiDimensionCodimWrapper at some point. Can you create an issue, for this.

Reply:
Because the indices of the elements are non-unique for every geometry type in the indexSets.

Reply:
So if you have three triangles and two quads, you would get:

```cpp
indexSet.index(triangle1) -> 0
indexSet.index(triangle2) -> 1
indexSet.index(triangle3) -> 2
indexSet.index(quad1) -> 0
indexSet.index(quad2) -> 1
```
So all this breaks if your grid has elements with different geometry types.

Reply:
The MultiDimensionCodimWrapper takes care of this and would return something like 

```cpp
mdcm.index(triangle1) -> 0
mdcm.index(triangle2) -> 1
mdcm.index(triangle3) -> 2
mdcm.index(quad1) -> 3
mdcm.index(quad2) -> 4
```

This also breaks in the result function currently.

Reply:
Ah yes thats true, I remember our discussion about that